src/scripts/toc-utils.ts

```
@@ -0,0 +1,105 @@
+    return acc;
+  }, {} as Record<number, number>);
+}
+
+/**
+ * Vérifie si un article a suffisamment de contenu pour une table des matières
+ * @param headings - Array des headings
+ * @param minHeadings - Nombre minimum de headings requis (défaut: 3)
+ * @returns true si l'article mérite une table des matières
+ */
+export function shouldShowToc(headings: AstroHeading[], minHeadings: number = 3): boolean {
+  return Boolean(headings && headings.length >= minHeadings);
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)